python/librmm/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
+
+include(../../rapids_config.cmake)
+
+project(
+  librmm-python
+  VERSION "${RAPIDS_VERSION}"
+  LANGUAGES CXX)
+
+# Check if rmm is already available. If so, it is the user's responsibility to ensure that the CMake
+# package is also available at build time of the Python rmm package.
+find_package(rmm "${RAPIDS_VERSION}")
+
+if(rmm_FOUND)
+  return()
+endif()
+
+unset(rmm_FOUND)
+
+set(BUILD_TESTS OFF)
+set(BUILD_BENCHMARKS OFF)
+set(CUDA_STATIC_RUNTIME ON)
+
+add_subdirectory(../.. rmm-cpp)
```

Comment:
*question*: The toplevel `CMakeLists.txt` says:
```
# cudart can be statically linked or dynamically linked the python 
# ecosystem wants dynamic linking
option(CUDA_STATIC_RUNTIME "Statically link the CUDA runtime" OFF)
```

Is that comment now out of date?

Reply:
It is... sort of? This comment predates our wheels, and seems to be a general statement about what Python wants. It also isn't quite true. I'll update it. We'll eventually want to statically link cudart everywhere anyway I think.